scapy/utils.py

```diff
@@ -1802,7 +1802,12 @@ def _read_options(self, options):
                         "%d !" % len(options))
                 raise EOFError
             if code != 0 and 4 + length <= len(options):
-                opts[code] = options[4:4 + length]
+                if code in [1, 2988, 2989, 19372, 19373]:  # https://www.ietf.org/archive/id/draft-tuexen-opsawg-pcapng-05.html#name-options-format
+                    if code not in opts:
+                        opts[code] = []
+                    cast(List[bytes], opts[code]).append(options[4:4 + length])
```

Comment:
I am not a big fan of this cast call. Can you remove it?

Reply:
I removed it 👍